Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+//https://github.com/feedhenry/fh-pipeline-library
+
+fhBuildNode {
+    stage('Install Dependencies') {
+        dir('fh-messaging') {
+            npmInstall {}
+        }
+        dir('fh-metrics') {
+            npmInstall {}
+        }
+    }
+
+    stage('Build') {
+        dir('fh-messaging') {
+            gruntCmd {
```

Comment:
Would `gruntBuild` not work here? If not, would it be useful to update the `cmd` below to be `fh:dist --only-bundle-deps`? Otherwise it'll build both artifacts, including the one without dependencies (we don't use that AFAIK).

Same questions for `fh-metrics` below.